### PR TITLE
Update: Peatio and Barong abilities

### PR DESCRIPTION
**Pull request by pkucherenk0**

### CODE REVIEW

**Comment by pkucherenk0** [comment]
to do:
1. update component versions
2. update abilities & permissions
3. amend commits

File: config/app.yml [hunk 1]
```diff
@@ -28,13 +28,13 @@ kyc:
...
-  peatio_matching_token: changeme
-  barong_token: changeme
-  finex_engine_token: changeme
+  root_token: s.Ei5sHdrRzFcrpVJi16s6qk1T
+  peatio_rails_token: s.U9G1R9X7S7PPmhN8UQ8nKqHg
+  peatio_crypto_token: s.AS632rUQH4NmF7mqngcArC4t
+  peatio_upstream_token: s.5XjP68T1z1O01shh8Gz2QfVB
+  peatio_matching_token: s.rWPjv4K5PBzUShssgqFlf69p
+  barong_token: s.P9m8LsVev4ClLXzof9ypkSqd
+  finex_engine_token: s.qadRclH9wmGZc2uXhOvKXmwG
```
Comment (dpatsora) [inline config/app.yml:37]: remove

File: templates/config/peatio/abilities.yml.erb [hunk 2]
```diff
@@ -97,81 +127,64 @@ admin_permissions:
...
+  sa_maker:
+    read:
+      - Operations::Account
+      - Order
+      - Trade
+      - StatsMemberPnl
+    create:
+      - Order
+    update:
+      - Order
```
Comment (dpatsora) [inline templates/config/peatio/abilities.yml.erb:190]: add new line

File: templates/config/barong/seeds.yml.erb [hunk 3]
```diff
@@ -188,5 +337,64 @@ permissions:
...
+ - { role: sa_maker, verb: all, path: api/v2/barong/resource/users/password, action: drop }
+ - { role: sa_maker, verb: all, path: api/v2/peatio/account/deposit_address, action: drop }
+ - { role: sa_maker, verb: all, path: api/v2/peatio/account/withdraws, action: drop }
+ - { role: sa_maker, verb: all, path: api/v2/peatio/account/beneficiaries, action: drop }
+ # Finex upstream access
+ - { role: sa_maker, verb: get, path: api/v2/open_finance, action: accept }
+ # ranger access
+ - { role: sa_maker, verb: get, path: api/v2/ranger, action: accept }
+ # applogic access
+ - { role: sa_maker, verb: all, path: api/v2/applogic, action: accept }
```
Comment (dpatsora) [inline templates/config/barong/seeds.yml.erb:400]: new line

**Review by dpatsora — APPROVED** [review]

**Pull request merged**